packages/cashier/build/constants.js

```diff
@@ -23,6 +23,7 @@ const ALIASES = {
     Config: path.resolve(__dirname, '../src/config'),
     Containers: path.resolve(__dirname, '../src/containers'),
     Constants: path.resolve(__dirname, '../src/constants'),
+    Pages: path.resolve(__dirname, '../src/pages'),
```

Comment:
@vinu-deriv maybe it's better to use an alphabetic order for ALIASES object? i mean add `Pages` after `Images`?